docs/docsite/rst/network/getting_started/first_playbook.rst

```diff
@@ -64,7 +64,7 @@ The flags in this command set seven values:
   - the module (-m, the Ansible module to run, using the fully qualified collection name (FQCN))
   - an extra variable ( -e, in this case, setting the network OS value)
 
-NOTE: If you use ``ssh-agent`` with ssh keys, Ansible loads them automatically. You can omit ``-k`` flag.
+NOTE: If you use ``ssh-agent`` with ssh keys, Ansible loads them automatically. You can omit ``-k`` flag. You can also use ``-e "ansible_ssh_pass=ansible"`` to include the password in the command without prompting the user.
```

Comment:
I can definitely see someone pointing this out as a bad recommendation from a security perspective.  What was driving this?

Reply:
ah wasn't  thinking of the security part. Just thinking it already has the -k option listed, but  that doesn't work afaik with ansible-navigator so documenting how to do the same thing w/o a prompt. Feel free to close if this is bad practice.  What I wanted to avoid is introducing the whole vault  thing in a 'getting started' guide.